test_reporting/sai_swss_invocations.py

```diff
@@ -0,0 +1,338 @@
+import json
+import os
+import pathlib
+from pathlib import Path
+
+from datetime import datetime
+
+from os import listdir, name
+from os.path import isfile, join
+import shutil
+
+from report_data_storage import KustoConnector
+
+'''
+for more detail, you can see
+https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/373699/sairedis-log-scanner
+'''
+
+
+ngsdevice_type = "ToRRouter"
+sai_path = "/data/sonic-mgmt-int/SAI/inc/"
+json_log_path = "/data/sonic-mgmt-int/test_reporting/test"
+sai_obj_feature_map = {}
+swss_device_log_items = {
```

Comment:
Put those data into a yaml and upload and use it as a sample?
Same as others, need more comments